index.html

```diff
@@ -262,36 +265,23 @@ <h6>CANOPY</h6>
               </div>
             </div>
           </div>
-        </div>
-        <section class="about-myself">
-          <div class="top">
-            <div class="primary-text-5">
-              <h1>About <br>
-              Myself</h1>
-            </div>
-              <p class="primary-text">I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
-            <div class="social-media">
-              <div class="supporting-text">
-                <span>LET'S CONNECT!</span>
-              </div>
-            </div>
-            <div class="social-media-icons">
-              <ul class="social-media-icons-list">
-                <li><a href="#"><img src="assets/twitter.svg" alt="Twitter" width="20" height="20"></a></li>
-                <li><a href="#"><img src="assets/linkedin.svg" alt="Linkedin" width="20" height="20"></a></li>
-                <li><a href="#"><img src="assets/m.svg" alt="Medium" width="20" height="16"></a></li>
-                <li><a href="#"><img src="assets/git.svg" alt="GitHub" width="18" height="20"></a></li>
-                <li><a href="#"><img src="assets/finger.svg" alt="Peace" width="15" height="20"></a></li>
-              </ul>
-            </div>
-            <div class="normal-button-secondary">
-              <div class="get-my-resume">
-                <button>Get my resume</button>
-              </div>
-            </div>
+        </section>
+        <section class="about-myself" id="about">
```

Comment:
- [ ] [OPTIONAL] The element in this section are not aligned properly kindly align the ``` about ``` section as in the Figma design, this will make your design more professional and clean

|Your design|Figma design|
|-|-|
|![Screenshot_20220604_091226](https://user-images.githubusercontent.com/25227022/171991243-e461e759-0a90-4374-b55b-1561f82d45c9.png)|![Screenshot_20220604_091649](https://user-images.githubusercontent.com/25227022/171991250-58d85a76-6c7a-490b-86be-de821bdafb22.png)|